src/marqo/core/unstructured_vespa_index/unstructured_document.py

```diff
@@ -52,15 +52,27 @@ class UnstructuredVespaDocument(MarqoBaseModel):
     id: str
     fields: UnstructuredVespaDocumentFields = Field(default_factory=UnstructuredVespaDocumentFields)
 
+    # For hybrid search
+    raw_tensor_score: float = None
+    raw_lexical_score: float = None
+
     _VESPA_DOC_FIELDS = "fields"
     _VESPA_DOC_ID = "id"
 
     @classmethod
     def from_vespa_document(cls, document: Dict) -> "UnstructuredVespaDocument":
         """Instantiate an UnstructuredVespaDocument from a Vespa document."""
         fields = document.get(cls._VESPA_DOC_FIELDS, {})
+
+        raw_lexical_score = fields.pop(unstructured_common.VESPA_DOC_HYBRID_RAW_LEXICAL_SCORE) \
```

Comment:
Does the custom searcher add this as a document field?

Reply:
yes. and it already does this for the structured implementation.